src/main/kotlin/no/nav/syfo/sykmelding/serviceuser/api/SykmeldingServiceuserApi.kt

```diff
@@ -24,6 +25,17 @@ fun Route.registrerSykmeldingServiceuserApiV1(sykmeldingerService: SykmeldingerS
                     call.respond(sykmelding)
                 }
             }
+            get("/sykmeldinger") {
```

Comment:
Jeg regner med at de har ting på stell, men kan du bare dobbeltsjekke med team risk at de faktisk har hjemmel til å hente ut sykmeldinger på vilkårlige brukere uten noe mer tilgangskontroll enn at det er en autentisert servicebruker? 

Sjekk gjerne også med dem at det ikke er en saksbehandler/veileder som trigger kallet, for hvis det er det burde vi ha sjekket at saksbehandleren har tilgang til personene de vil hente sykmeldinger for.

Reply:
👍 Jeg sjekker! :)